snow/engine/snowman/bootstrap/bootstrapper.go

```diff
@@ -587,8 +587,8 @@ func (b *bootstrapper) checkFinish(ctx context.Context) error {
 	// Note that executedBlocks < c*previouslyExecuted ( 0 <= c < 1 ) is enforced
 	// so that the bootstrapping process will terminate even as new blocks are
 	// being issued.
-	if b.Config.RetryBootstrap && executedBlocks > 0 && executedBlocks < previouslyExecuted/2 {
-		return b.Restart(ctx, true)
+	if executedBlocks > 0 && executedBlocks < previouslyExecuted/2 {
```

Comment:
This change is why test tests needed to be updated (because `b.Config.RetryBootstrap` was `false` in a number of tests previously)